strawberryfields/engine.py

```diff
@@ -490,18 +481,21 @@ class RemoteEngine:
         target (str): the target device
         connection (strawberryfields.api.Connection): a connection to the remote job
             execution platform
+        backend_options (Dict[str, Any]): keyword arguments for the backend
     """
 
     POLLING_INTERVAL_SECONDS = 1
     VALID_TARGETS = ("X8_01", "X12_01", "X12_02")
+    DEFAULT_TARGETS = {"X8": "X8_01", "X12": "X12_01"}
 
-    def __init__(self, target: str, connection: Connection = Connection()):
+    def __init__(self, target: str, connection: Connection = Connection(), backend_options: dict = None):
         if target not in self.VALID_TARGETS:
             raise ValueError(
                 "Invalid engine target: {} (valid targets: {})".format(target, self.VALID_TARGETS)
```

Comment:
What if `target` is a key in `DEFAULT_TARGETS` but not in `VALID_TARGETS`, like `X8` and `X12`? Shouldn't these also be valid targets (since they are changed to the default target further down)?

Reply:
Ah, good catch! I've fixed this by moving the line
```python
self._target = self.DEFAULT_TARGETS.get(target, target)
```
here instead of in `eng.run()`.

It also makes a lot more sense to have it here, since that logic is independent of program execution, and didn't need to be called each time.